spec/requests/rounds_management_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Rounds management', type: :request do
+  describe 'POST teams/:id/rounds' do
+    let(:team) { create(:team) }
+    let(:user) { create(:user) }
+
+    subject { post team_rounds_path(team_id: team.id, as: user) }
+
+    before do
+      user.teams << team
+    end
+
+    it "creates a round and redirects to the round's show page" do
```

Comment:
"pseudo-suggestion" (needs validation)
```suggestion
        context 'Authenticated user belongs to team round is created with' do
      let(:user) { create(:user) }
      let(:team) { create(:team) }
      before { user.teams << team }

      it "creates a round for with user and team and redirects to the round's show page" do
        post team_rounds_path(team_id: team.id, as: user)

        round = Round.find_by(user: user, team: team)

        expect(round).to be_present
        expect(response).to have_http_status(:found)
        expect(response).to redirect_to team_round_path(team_id: round.team.id, id: round.id)
        expect(flash[:notice]).to eq 'Round created successfully.'
      end
    end
```